### PR TITLE
added rxAndroid to add persistent data to app

### PR DESCRIPTION
**Pull request by HeribertoSanchez93**
new feature about data persist and get data from cache despite app  does not have connection internet

### CODE REVIEW

File: app/src/main/java/com/example/bootcampproject/data/local/AvailableBooksDao.kt [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import androidx.room.Update
 import com.example.bootcampproject.data.mock.AvailableBook
 
-
 @Dao
 interface AvailableBooksDao {
     @Query("Select * From AvailableBook")
```
Comment (cefmmaku) [inline app/src/main/java/com/example/bootcampproject/data/local/AvailableBooksDao.kt]: You can store table names in a constants file.

File: app/src/main/java/com/example/bootcampproject/data/local/DateUpdateDao.kt [hunk 2]
```diff
@@ -5,7 +5,6 @@ import androidx.room.OnConflictStrategy
 import androidx.room.Query
 import androidx.room.Update
 import com.example.bootcampproject.data.mock.DateUpdate
-import com.example.bootcampproject.domain.Currency
 
 interface DateUpdateDao {
     @Query("Select * From DateUpdate")
```
Comment (cefmmaku) [inline app/src/main/java/com/example/bootcampproject/data/local/DateUpdateDao.kt:10]: You can also use a constants file

File: app/src/main/res/layout/item_asks.xml [hunk 3]
```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="100dp"
```
Comment (cefmmaku) [inline app/src/main/res/layout/item_asks.xml]: Use dimens.xml resource files

File: app/src/main/res/layout/item_currency.xml [hunk 4]
```diff
@@ -22,7 +22,7 @@
             android:text="@string/textview"
             android:textAllCaps="false"
             android:textColor="#D4D4A9"
-            android:textSize="20sp"
+            android:textSize="34sp"
```
Comment (cefmmaku) [inline app/src/main/res/layout/item_currency.xml]: Use dimens.xml resource files

**Review by cefmmaku — CHANGES_REQUESTED** [review]
Small changed pending.

File: app/src/main/res/layout/item_currency.xml (hunk 4 above)
Comment (HeribertoSanchez93) [reply app/src/main/res/layout/item_currency.xml]: done

File: app/src/main/res/layout/item_asks.xml (hunk 3 above)
Comment (HeribertoSanchez93) [reply app/src/main/res/layout/item_asks.xml]: done

File: app/src/main/java/com/example/bootcampproject/data/local/DateUpdateDao.kt (hunk 2 above)
Comment (HeribertoSanchez93) [reply app/src/main/java/com/example/bootcampproject/data/local/DateUpdateDao.kt:10]: done

File: app/src/main/java/com/example/bootcampproject/data/local/AvailableBooksDao.kt (hunk 1 above)
Comment (HeribertoSanchez93) [reply app/src/main/java/com/example/bootcampproject/data/local/AvailableBooksDao.kt]: done

**Pull request merged**